### ISSUE TITLE
Where do the python packages get installed?

### ISSUE THREAD

**Issue by wgordon17** [pre-commit/pre-commit#1155]
Specifically, I'm using the pylint hook (https://github.com/pre-commit/pre-commit/issues), and I can't for the life of me figure out how I can run `pylint` directly. I need to know where the `pylint` binary is for PyCharm integration.

**Comment by asottile** [comment]
it's hidden implementation detail, I wouldn't realllly depend on it -- you can use `pre-commit run pylint --files` as a proxy to the executable

if you want the full answer though, they live in `~/.cache/pre-commit/repo*/py_env*/bin/...`

also looks like your link isn't the one you intended

**Comment by wgordon17** [comment]
Similarly, I've noticed that if I run `pre-commit run pylint` it doesn't seem to find my installed modules

    Unable to import 'django.apps' (import-error)

But if I run a locally installed version of `pylint` it has no issues. I'm assuming it's because `pre-commit` is using its own, separate packages. What's the correct way to _let_ `pre-commit` see the same packages I have installed in my project virtualenv?
